src/components/movieList/MovieList.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import MovieListItem from '../movieListItem/MovieListItem';
+import Spinner from '../spinner/Spinner';
+import styles from './MovieList.module.scss';
+
+const MovieList = ({ films, genres }) => {
+  const isFilmLoaded = useSelector((state) => state.loadingFilms);
+  const movieCards = films
+    ? films.map((movie) => (
+      <li key={movie.id} className={styles.movieListItem}>
+        <MovieListItem movie={movie} genres={genres} />
+      </li>
+    ))
+    : null;
```

Comment:
I don't know if you are awared about pattern like this.
```
const MovieList = ({ films = [], genres }) => {
  ...
  const movieCards = films.map((movie) => (
      <li key={movie.id} className={styles.movieListItem}>
        <MovieListItem movie={movie} genres={genres} />
      </li>
    ));
```
It render nothing if there is no films.
You can choose an approacah which you like more.